webscrapers/BCB_IF/Bacen_IF_Data.py

```diff
@@ -0,0 +1,60 @@
+# Quantitative Finance
```

Comment:
It's great that you guys are writing an example file, but it should not have ´input´s along the way.